internal/config/config.go

```diff
@@ -9,6 +9,10 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+func New() *Config {
+	return &Config{}
```

Comment:
we can also use a global var as it's usually the case for Config objects, and then call Watch when we need to

Reply:
Global variables should be avoided.